bin/dev/vagrant_provision.sh

```diff
@@ -120,10 +116,6 @@ bin/console user password "$ADMIN_NICK" "$ADMIN_PASSW"
 bin/console user add "$USER_NICK" "$USER_NICK" "$USER_NICK@friendica.local" en
 bin/console user password "$USER_NICK" "$USER_PASSW"
 
-# set the admin
-bin/console config config admin_email ""$ADMIN_NICK@friendica.local""
```

Comment:
why not set `admin` user as admin anymore?

Reply:
When you use the predefined config file, the admin is set there. Setting it here again will lead to an error message in the Vagrant console.